public/IMG/bank-artwork.svg

```diff
@@ -0,0 +1 @@
112,5.238,2.237,8.09a27.755,27.755,0,0,0,23.449,8.929c3.728-.6,6.151-2.218,7.046-4.567s.112-5.238-2.218-8.108a28.277,28.277,0,0,0-20.374-9.152Z" transform="translate(40.406 126.599)" fill="#150035"/><path d="M72.254,155.672a22.592,22.592,0,0,0-16.776-6.412c-5.592.578-7.009,4.4-3.243,8.519a22.61,22.61,0,0,0,16.925,6.431C74.659,163.706,76.113,159.81,72.254,155.672Z" transform="translate(43.313 128.945)" fill="#150035"/><path d="M68.046,165.487a23.934,23.934,0,0,1-16.068-6.766c-2.05-2.237-2.777-4.53-2.05-6.449s2.8-3.132,5.816-3.448a23.58,23.58,0,0,1,17.6,6.692h0c2.05,2.237,2.777,4.53,2.05,6.431s-2.8,3.15-5.8,3.467C69.089,165.449,68.568,165.487,68.046,165.487ZM57.327,150.369a11.609,11.609,0,0,0-1.417,0c-2.349.242-3.952,1.118-4.455,2.4a5,5,0,0,0,1.734,4.772,21.772,21.772,0,0,0,16.217,6.17c2.349-.242,3.97-1.137,4.455-2.423a4.958,4.958,0,0,0-1.715-4.753h0a22.089,22.089,0,0,0-14.819-6.17Z" transform="translate(42.899 128.563)" fill="#150035"/><path d="M71.387,157.045a22.592,22.592,0,0,0-16.925-6.412,6.953,6.953,0,0,0-4.362,1.752,6.99,6.99,0,0,0,2.088,4.213,22.611,22.611,0,0,0,16.925,6.431,6.953,6.953,0,0,0,4.362-1.752A7.027,7.027,0,0,0,71.387,157.045Z" transform="translate(43.285 130.125)" fill="#b288d3"/><path d="M68.031,164.3a23.934,23.934,0,0,1-16.068-6.766A7.792,7.792,0,0,1,49.67,152.8v-.354l.242-.242a7.717,7.717,0,0,1,4.865-2.013,23.6,23.6,0,0,1,17.6,6.692,7.773,7.773,0,0,1,2.311,4.735v.336l-.242.261a7.661,7.661,0,0,1-4.846,2.013C69.074,164.265,68.553,164.3,68.031,164.3ZM51.254,153.119a6.617,6.617,0,0,0,1.864,3.355,21.772,21.772,0,0,0,16.217,6.17,6.71,6.71,0,0,0,3.6-1.286,6.748,6.748,0,0,0-1.864-3.337h0a21.734,21.734,0,0,0-16.236-6.17,6.71,6.71,0,0,0-3.5,1.23Z" transform="translate(42.914 129.746)" fill="#150035"/><rect width="3.989" height="1.249" transform="translate(109.891 301.372) rotate(-69.2)" fill="#b288d3"/><rect width="3.989" height="1.249" transform="translate(86.888 289.023) rotate(-69.2)" fill="#b288d3"/><rect width="2.572" height="1.249" transform="translate(107.901 282.573) rotate(-69.2)" fill="#b288d3"/><path d="M52.523,151.108l-.783,2.05,2.349,2.088,2.144-1.659h0l7.754,9.525a17.184,17.184,0,0,0,2.74.112l1.3-.2.783-2.05-11-10.569Z" transform="translate(44.702 129.949)" fill="#150035"/><path d="M66.513,164.181a16.661,16.661,0,0,1-2.125-.131h-.242l-7.568-9.32-2.088,1.622-3.15-2.684,1.062-2.8,6.021-.8,11.594,11.054L68.974,163.9l-1.678.261ZM64.8,162.839a14.783,14.783,0,0,0,2.386,0l.876-.13.485-1.323-10.55-10.1-4.679.615-.485,1.286,1.715,1.491,1.864-1.435h.522l.28.28Z" transform="translate(44.357 129.663)" fill="#150035"/><path d="M56.747,152.495h0l-2.144,1.64L52.18,152.01l3.02-2.237,3.094-.5,10.327,12.713h0l-3.635.6Z" transform="translate(45.082 128.972)" fill="#fff"/><path d="M65.153,163.535l-8.034-9.842-2.088,1.622-3.411-2.982,3.728-2.8,3.616-.6L69.776,162.1l-.28.671Zm-7.7-11.352.261.261,7.941,9.656,2.218-.354-9.32-11.52-2.591.429-2.274,1.678,1.454,1.268,1.864-1.417Z" transform="translate(44.598 128.687)" fill="#150035"/></g></g></svg>
```

Comment:
- Put all images in this path: ```public/assets/images/```
- Put all fonts in this path: ```public/assets/fonts/```

Reply:
solved